iso15118/shared/settings.py

```diff
@@ -2,6 +2,15 @@
 
 import environs
 
+
+class SettingKey:
+    PKI_PATH = "pki_path"
+    MESSAGE_LOG_JSON = "message_log_json"
+    MESSAGE_LOG_EXI = "message_log_exi"
+    V20_SERVICE_CONFIG = "v20_service_config"
```

Comment:
If the constants are in upper case, then logging becomes consistent with what we have today. When I ran secc the env were logged like this:
```
DEBUG    2023-10-03 21:47:41,980 - asyncio (54): Using selector: KqueueSelector
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (107): SECC settings:
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (109): pki_path                      : /Users/shalinnijel/Desktop/Switch/repos/josev_pro_repos/josev_pro_release/iso15118_service/iso15118_service/pki/
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (109): message_log_json              : True
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (109): message_log_exi               : False
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (109): v20_service_config            : /Users/shalinnijel/Desktop/Switch/repos/iso15118_repos/iso15118_release_test/iso15118/shared/examples/secc/15118_20/service_config.json
INFO    2023-10-03 21:47:41,981 - iso15118.secc.secc_settings (109): enable_tls_1_3                : False
INFO    2023-10-03 21:47:41,982 - iso15118.secc.secc_settings (111): NETWORK_INTERFACE             : en0
INFO    2023-10-03 21:47:41,982 - iso15118.secc.secc_settings (111): LOG_LEVEL                     : DEBUG
```

Reply:
I see that in secc_settings, while logging, the env variables are automatically picked up - but here, we could still define the constants in upper case and that would fix the logging, right? Anything else that might cause issues if we make them lowercase?

Reply:
Done